notifications/01-27-21-1500.md

```diff
@@ -0,0 +1,24 @@
+
+**What is happening?**
+
+Sysdig engineers will be performing scheduled maintenance in our SaaS environments. Specifically, they will be streamlining our database architecture for more resiliency and better performance.
+
+Updates will be provided on sysdig.statuspage.io throughout the maintenance operation.
+
+**When?**
+
+Our SaaS region will be worked on Sunday Feb 3rd at 06:00
```

Comment:
```suggestion
Our SaaS region will be worked on Wednesday Feb 3rd at 06:00
```

Reply:
Derp, us-west was Sunday Feb 8th and I got that stuck in my head. We are us-east.